omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ClobDatatypeStorageControllerTest.java

```diff
@@ -0,0 +1,87 @@
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs1_9;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.openmrs.module.webservices.rest.web.RestTestConstants1_9;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.ClobDatatypeStorageController;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.openmrs.util.DatabaseUpdater;
+import org.openmrs.util.OpenmrsUtil;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.mock.web.MockMultipartHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.ResultSet;
+
+public class ClobDatatypeStorageControllerTest extends MainResourceControllerTest {
+
+    @InjectMocks
+    private ClobDatatypeStorageController clobController;
+
+    @Before
+    public void before() throws Exception{
+        executeDataSet(RestTestConstants1_9.FORM_RESOURCE_DATA_SET);
+    }
+
+    @Test
+    public void create_shouldAcceptAndStoreClobDataViaPost() throws Exception{
+        long before = getAllCount();
+        System.out.println(before);
+
+        File file = new File("omod-1.9/src/test/resources/formResourcefile.txt");
+        Assert.assertTrue(file.exists());
+
+        MockMultipartFile toUpload = new MockMultipartFile("value", "formresource.txt", "text/plain",
```

Comment:
My understanding is that "value" is actually the name of the parameter that needs to match with the controller @RequestParam